### PR TITLE
[DOC] contributing page: concrete initial steps

### PR DESCRIPTION
**Pull request by fkiraly**
This PR updates `contributing.rst` with the concrete first steps from pinned issue #1147.

### CODE REVIEW

File: docs/source/get_involved/contributing.rst [hunk 1]
```diff
@@ -22,14 +22,13 @@ For a detailed overview of current and future work, check out our :ref:`roadmap`
...
-   `Guide for Reproducible
-   Research <https://the-turing-way.netlify.app/reproducible-research/reproducible-research.html>`_ to get started and learn more about open-source collaboration.
+Recommended steps for first time contributors, or to get started with regular contributions:
+
+1. Say hello and get set up for development, see `instructions in the developer guide<https://www.sktime.org/en/stable/developer_guide.html>`_
+2. Pick a good first issue to work on, see a collection `in this summary issue <https://github.com/alan-turing-institute/sktime/issues/1147>`_ , or from `this list <https://github.com/alan-turing-institute/sktime/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22>`_
+   suggestion: pick something small with simple content to learn the “process”
+3. Feel free to attend the regular community collab sessions or one of the topic specific stand-ups and tech sessions (see schedule on `Slack`_)
+4. Once your first PR is merged and you’ve seen how things work, think about your regular time commitment. Optionally, continue attending the Friday community collaboration sessions and stand-ups; or, optionally, `apply for mentoring<mentoring>`_
```
[the remark this answers is not in the document]
Comment (fkiraly) [reply docs/source/get_involved/contributing.rst]: I've changed point 4 to more optional phrasing, is that better?
the "think about goals" is perhaps better in mentoring week 0

[the remark this answers is not in the document]
Comment (GuzalBulatova) [reply docs/source/get_involved/contributing.rst]: I think it sounds better, thanks!

**Comment by fkiraly** [comment]
(do you want to approve or request changes, @GuzalBulatova?)

**Review by GuzalBulatova — APPROVED** [review]
Thanks Franz, looks all good 👍

**Pull request merged**